thrall/app/ThrallComponents.scala

```diff
@@ -53,9 +55,10 @@ class ThrallComponents(context: Context) extends GridComponents(context, new Thr
 
   val highPrioritySource: Source[KinesisRecord, Future[Done]] = KinesisSource(highPriorityKinesisConfig)
   val lowPrioritySource: Source[KinesisRecord, Future[Done]] = KinesisSource(lowPriorityKinesisConfig)
```

Comment:
we should rename these to reflect their purpose, rather than their priority, e.g...
```suggestion
  val userActionsSource: Source[KinesisRecord, Future[Done]] = KinesisSource(highPriorityKinesisConfig)
  val backgroundTasksSource: Source[KinesisRecord, Future[Done]] = KinesisSource(lowPriorityKinesisConfig)
```